docs/semantics/string.md

```diff
@@ -0,0 +1,38 @@
+# String Semantics
+
+## Overview
+
+The string type encapsulates sequences of characters. Strings are declared via the `str` keyword. 
+
+## Syntax
+
+A string type declaration in general takes the form of `str myString = "Text";`.  
+
+## Semantics
+
+Initalized string objects are immutable. Future support for a mutable string class, in the manner of `StringBuilder` in Java, may be supported in our standard library. 

```

Comment:
We should talk to today about this and how we're going to store strings

Reply:
Agreed; though I'm assuming we should leave discussion of how data types are implemented out of our semantics docs?